compiler/noirc_frontend/src/locations.rs

```diff
@@ -33,10 +33,21 @@ impl NodeInterner {
         match reference {
             ReferenceId::Module(id) => self.module_location(&id),
             ReferenceId::Function(id) => self.function_modifiers(&id).name_location,
-            ReferenceId::Struct(id) => self.struct_location(&id),
-            ReferenceId::Trait(id) => self.trait_location(&id),
```

Comment:
Why are these no longer `struct_location` or `trait_location`? I'm guessing those held the location of the entire struct/trait and we want to navigate to only the name?

Are those methods still used?

Reply:
I actually introduced those two in a previous PR to track the struct/trait name location. In this PR I realized they were already captured, but somewhere else 😊 . So now those locations are used and there's no need to track them separately :-)